test/sneaker_handlers/retry_handler_test.rb

```diff
@@ -0,0 +1,54 @@
+require "test_helper"
+require "sneakers"
+require "sneakers/runner"
+require "rabbitmq/http/client"
+require 'pry'
```

Comment:
Looking through the whole list I don't think we need a bunch of these anymore. Could you give them a quick audit>?